VocalRaidAssistant.lua

```diff
@@ -65,7 +82,13 @@ function VRA:ChangeProfile()
 end
 
 function VRA:ChatCommand()
-	self.ACD:Open("VocalRaidAssistantConfig")
+	if VRAOptionsFrame and VRAOptionsFrame:IsShown() then
```

Comment:
simply ask ACD:
if self.ACD.OpenFrames["VocalRaidAssistantConfig"]
Then you can remove the "trigger" lines, too.

Reply:
Smart, fixed